src/ur/ur_driver.cpp

```diff
@@ -361,8 +361,10 @@ bool UrDriver::startForceMode(const vector6d_t& task_frame, const vector6uint32_
 {
   if (robot_version_.major < 5)
   {
-    URCL_LOG_ERROR("Force mode gain scaling factor cannot be set on a CB3 robot.");
-    return false;
+    std::stringstream ss;
+    ss << "Force mode gain scaling factor cannot be set on a CB3 robot.";
+    URCL_LOG_ERROR(ss.str().c_str());
+    throw std::invalid_argument(ss.str().c_str());
```

Comment:
I don't think `std::invalid_argument` is the correct exception here. Sorry, that I didn't specify this earlier. I meant exceptions in general, not an invalid_argument exception for everything.
It probably would be good to create a `IncompatibleRobotVersion` exception in [URExceoptions](https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/master/include/ur_client_library/exceptions.h) Or maybe use a `VersionMismatch` here?